src/main/java/com/salesforce/dataloader/controller/Controller.java

```diff
@@ -146,8 +147,11 @@ private String getConfigDirFromArgMap(String[] args) {
                 argNameValuePair.get(CONFIG_DIR_PROP) : null;
     }
 
-    public static void initStaticVariable() throws ControllerInitializationException {
-        Properties versionProps = new Properties();
+    private static synchronized void initStaticVariable() throws ControllerInitializationException {
+    	if (areStaticVarsInitialized) {
```

Comment:
Indentation

Reply:
Not sure if we need `synchronized` here as DL is a single threaded app

Reply:
Better be (thread) safe :).